src/classes/AccountTriggerHandler.cls

```diff
@@ -113,4 +113,4 @@ public void UpdateLastTransactionDate(Account[] updatedSUs){
         }
     
```

Comment:
Methods should use the `lowerCamelCase` naming convention. I suggest to rename accordingly.